lib/bolt/project.rb

```diff
@@ -1,11 +1,13 @@
 # frozen_string_literal: true
 
 require 'pathname'
+require 'bolt/pal'
 
 module Bolt
   class Project
     BOLTDIR_NAME = 'Boltdir'
     PROJECT_SETTINGS = {
+      "name"  => "The name of the project",
       "plans" => "An array of plan names that can be included in `bolt plan show` output",
       "tasks" => "An array of task names that can be included in `bolt task show` output"
```

Comment:
I think we should rephrase slightly so that it's clear to users that these options whitelist plan and task names
```suggestion
      "plans" => "An array of plan names to whitelist. Whitelisted plans are included in `bolt plan show` output",
      "tasks" => "An array of task names to whitelist. Whitelisted tasks are included in `bolt task show` output"
```

Reply:
Updateddddd